charts/data-prepper/CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.1.2]
```

Comment:
@janhoy thanks for the PR, the releases are all incremental. Since this PR https://github.com/opensearch-project/helm-charts/pull/631 is merged, can you please update this to `[0.2.1`] ?

Reply:
Following up here, @janhoy please take a look we can get this merged.
Thanks 
@getsaurabh02